front/src/modules/favorites/components/Favorites.tsx

```diff
@@ -28,32 +28,28 @@ export function Favorites() {
             <NavItem
               key={id}
               label={`${person.firstName} ${person.lastName}`}
-              icon={
-                <Avatar
-                  key={id}
-                  colorId={person.id}
-                  avatarUrl={person.avatarUrl ?? ''}
-                  type="rounded"
-                  placeholder={`${person.firstName} ${person.lastName}`}
-                  size="md"
-                />
-              }
+              Icon={Avatar}
```

Comment:
Same as CommandMenuItem, maybe we can find a shared to pass `IconComponent | AvatarProps` ?